### PR TITLE
dcaf-abe documentation

### PR DESCRIPTION
**Pull request by adean20csc**
This is some cleanup and some new doc.

We were using AutoDeployNode all over the place and Automation in less.
Since autodeploy is also in the directory structure and referred to more thought make more sense to stay consistent.

### CODE REVIEW

**Comment by mtnbikenc** [comment]
Doc review :+1:

File: docsite/source/ansible_offline.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+In some circumstances it might be useful to have a working copy of Ansible
+without having to resort to installing via an distro-package or python pip. To
+accomplish this see the following steps below.
 
 Prerequisites
 ~~~~~~~~~~~~~
 
-First we need to make sure python-pip is installed, if we are using
-CentOS or RHEL EPEL needs to be installed.
+First we need to make sure python-pip is installed, if we are using CentOS or
```
Comment (jdextercsc) [inline docsite/source/ansible_offline.rst]: I don't know why we want the Centos in here, we haven't tested against CentOS.

**Comment by mtnbikenc** [comment]
@adean20csc Please remove these files, they are legacy and not part of the docsite hierarchy.  This should remove some confusion.
```
docsite/source/ansible_offline.rst
docsite/source/bootstrap_method.rst
docsite/source/packer.rst
```

File: docsite/source/ansible_offline.rst (hunk 1 above)
Comment (mtnbikenc) [inline docsite/source/ansible_offline.rst]: This file was created when we were first getting the web based docs going, but is not applicable.  We can just pull it out.

**Comment by jdextercsc** [comment]
Doc review :+1:

**Pull request merged**